source/class/qx/html/Jsx.js

```diff
@@ -194,6 +195,11 @@ qx.Class.define("qx.html.Jsx", {
           tagname,
           newAttrs
         );
+        // if we're in a browser, create a dom element with the same tagname as the resulting `element`
```

Comment:
does make sense, but is this related to the change described ?

Reply:
I've removed the comment, it wasn't providing anything useful.

Creating and using a dom node immediately for jsx-created `Elements` is done so that the call to `qx.html.Element._copyData` is passed `true` for the parameter `fromMarkup`